packages/config/adders/community.ts

```diff
@@ -0,0 +1,17 @@
+export type CommunityAdder = {
+    id: string;
+    displayName: string;
+    description: string;
+    package: string;
+};
+
+export type CommunityAdders = CommunityAdder[];
+
+export const communityAdders: CommunityAdders = [
+    {
+        id: "test",
+        displayName: "Test",
+        description: "Test description",
+        package: "@test/setup",
+    },
```

Comment:
should this be here or is this a leftover during testing?

Reply:
Good catch, removed